src/plugins/src/meta_srv.rs

```diff
@@ -14,12 +14,12 @@
 
 use common_base::Plugins;
 use meta_srv::error::Result;
-use meta_srv::metasrv::MetaSrvOptions;
+use meta_srv::metasrv::MetasrvOptions;
 
-pub async fn setup_meta_srv_plugins(_opts: &mut MetaSrvOptions) -> Result<Plugins> {
+pub async fn setup_metasrv_plugins(_opts: &mut MetasrvOptions) -> Result<Plugins> {
     Ok(Plugins::new())
 }
 
-pub async fn start_meta_srv_plugins(_plugins: Plugins) -> Result<()> {
+pub async fn start_metasrv_plugins(_plugins: Plugins) -> Result<()> {
```

Comment:
Will there be any issue if I make this change? cc @shuiyisong

Reply:
It wouldn't be a compatibility issue 🚀